packages/amplify-codegen-e2e-core/src/init/initProjectHelper.ts

```diff
@@ -108,8 +108,7 @@ export function initAndroidProjectWithProfile(cwd: string, settings: Object): Pr
       .wait('Choose your default editor:')
       .sendLine(s.editor)
       .wait("Choose the type of app that you're building")
-      .send('j')
-      .sendCarriageReturn()
+      .sendLine('android')
```

Comment:
Is this related to the CLI version update?

Reply:
Yeah I think so. Bunch of e2es started failing post release, have fixed most.